src/guides/about-multiprotocol.html

```diff
@@ -25,19 +25,32 @@ <h2>About channels</h2>
   </span></p>
   <p><span>
     The wireless chip in Home Assistant Yellow supports all 16 channels. However,
-    it can only communicate on one single channel at a time. Due to this limitation
-    it is crucial that Zigbee and Thread run on the same channel for reliable
-    operation!
+    it can only communicate on one single channel at a time. Due to this limitation, it is crucial that Zigbee and Thread run on the same channel for reliable operation!
     
     Thanks to the PAN-ID, the Zigbee and Thread networks are still separated and
-    therefore won't interfere with each other.
+    therefore won't interfere with each other. <br>
+    <p>
+    The illustration below shows on which frequency range each channel operates. It also shows on which channels a specific protocol operates. 
+    <ul>
+      <li>Grey sections mark the channels used by Wi-Fi.</li>
+      <li>Blue sections mark the channels used by Bluetooth Low Energy.</li>
+      <li>Orange sections mark the channels used both by Zigbee and Thread.</li>
+    </ul>
+    As you can see, channels 15, 20, and 25 are used by Zigbee and Thread. These channels are less likely to be affected by Wi-Fi interference.
```

Comment:
```suggestion
    Zigbee/Thread channels 15, 20, and 25 are less likely to be affected by Wi-Fi interference since they are in between the most commonly Wi-Fi channels 1, 6 and 11.
```